colossalai/zero/low_level/low_level_optim.py

```diff
@@ -95,6 +92,11 @@ def __init__(
 
         self._cpu_offload = cpu_offload
 
+        # grad accumulation
+        self.require_grad_sync = True
```

Comment:
We can control grad accumulation by this attribute, instead of `grad_accumulate_interval`. This is consistent of torch DDP.

Reply:
Yes, in fact, I am considering refactoring the GA part with supporting no_sync in the plugin.
However, I would like the PR can be merged first :<

Reply:
ok